nuxt.config.js

```diff
@@ -54,6 +54,14 @@ export default {
     defaultLocale: "en",
   },
 
+  render: {
+    bundleRenderer: {
+      shouldPreload: (file, type) => {
+        return ['font'].includes(type)
+      }
+    }
+  },
+
```

Comment:
praise: excellent idea to pre-load the fonts to ensure a smooth user experience without any jumping layouts.

Reply:
question (blocking): We are dealing with the problem of FOUT here. Though afaik modern web browsers deal with this via a mix of FOUT and FOIT and we want to avoid this or the web page will feel slow.

Font preloading best practices suggest that we only pre-load the fonts that are needed for that specific page. More precisely: the fonts that are seen in the viewport for an optimal experience. We should limit the different types here. Everything else can be downloaded later. We should favor speed. Only pre-load the font for paragraphs and one headline (e.g. h1).

How many styles are we using in total? Are we downloading the font from a CDN or our server? I'd suggest switching to a CDN here to increase download speed. Make sure to set the crossorigin attribute then.